lang/en.json

```
@@ -18,6 +18,8 @@
         "Groups": "Campaign Groups",
         "Placements": "Campaign Placements",
         "CampaignIds": "Campaign Ids",
-        "CombinedSourcesMediums": "Campaign Source - Medium"
+        "CombinedSourcesMediums": "Campaign Source - Medium",
+        "DoNotChangeCaseOfUtmParametersTitle": "Retain campaign parameter capitalisation",
```

"Keep" might be a more commonly used word.
```suggestion
        "DoNotChangeCaseOfUtmParametersTitle": "Keep campaign parameter capitalization",
```